app/views/Dashboard/Country/index.tsx

```diff
@@ -382,147 +372,120 @@ function Country(props: Props) {
                             </ResponsiveContainer>
                         </ContainerCard>
-                        <ContainerCard
-                            className={styles.ageDisaggregation}
-                            contentClassName={styles.responsiveContent}
-                            heading="Age disaggregation"
-                            headerDescription="Lorem ipsum explaining the topic"
-                            headingSize="extraSmall"
-                        >
-                            <ResponsiveContainer className={styles.responsiveContainer}>
-                                <PieChart>
-                                    <Pie
-                                        data={countryResponse
-                                            ?.disaggregation.ageDisaggregation}
-                                        dataKey="indicatorValue"
-                                        labelLine={false}
-                                        cx={100}
-                                        cy={100}
-                                        outerRadius={70}
-                                    >
-                                        {age.map((entry) => (
-                                            <Cell
-                                                key={entry.age}
-                                                fill={entry.fill}
+                    )}
+                    {filterValues?.indicator && (
+                        <div className={styles.indicatorWrapper}>
+                            <PercentageStats
+                                className={styles.percentageCard}
+                                heading="Percentage of unvaccinated individuals who have tried to get vaccinated"
+                                headingSize="extraSmall"
+                                statValue={56}
+                                suffix="%"
+                                icon={null}
+                            />
+                            <IndicatorChart
+                                className={styles.indicatorsChart}
+                            />
+                            <ContainerCard
+                                className={styles.disaggregation}
+                                contentClassName={styles.disaggregationContent}
+                                heading="Disaggregation"
+                                headerDescription="Lorem ipsum explaining the topic"
+                                headingSize="extraSmall"
+                            >
+                                <div className={styles.genderDisaggregation}>
+                                    <div>Gender Disaggregation</div>
+                                    <ResponsiveContainer className={styles.responsiveContainer}>
+                                        <BarChart
+                                            data={genderDisaggregation}
+                                        >
```

Comment:
Should we check if the disaggregation data is defined and only show the bar chart of the existing values?